Resources/Prototypes/Entities/Objects/Materials/materials.yml

```
@@ -587,6 +589,10 @@
   - type: Appearance
   - type: Item
     heldPrefix: bananium
+  - type: Tag
+    tags:
+    - ConstructionMaterial    #Moffstation
+    - RawMaterial
```

```suggestion
  # Moffstation - Begin
  - type: Tag
    tags:
    - ConstructionMaterial
    - RawMaterial
  # Moffstation - End
```
Guards are needed around the entire modified syntax, not just the "logical" addition of the new tag, sadly.